examples/data_preprocessing/wiennerisches_diarium_historical_dataset/data_preprocessing_sterbelisten_wiener_diarum.ipynb

```diff
@@ -0,0 +1,1246 @@
+    "\n",
+    "**Z Matrices**\n",
+    "numpy.narray\n",
+    "columns= keywords \n",
+    "rows = words in sentence\n",
+    "\n",
+    "\n",
+    "**5. Building test data** \n",
+    "\n",
+    "manually checked! \n",
+    "There is the 'timemachine_evaluatuin_v1_edited_corrected.jsonl which should contain manually corrected place names but it is not usable for our purposes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Please check requirements are in the requirements.txt \n",
+    "\n",
+    "The code in this tutorial is also available as a data_preprocessing_wiener_diarum_toponym.py.\n",
+    "\n",
+    "This folder contains: \n",
+    "- data_preprocessing_sterbelisten_wiener_diarum.ipn\n",
+    "- data_preprocessing_wiener_diarum_toponym.py\n",
+    "- requirements.ipynb\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Imports**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import re\n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from typing import List, Tuple, Dict\n",
+    "\n",
+    "from tqdm import tqdm\n",
+    "from joblib import dump\n",
+    "from minio import Minio\n",
+    "client = Minio(\"knodle.cc\", secure=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'../../../data_from_minio/wiener_diarum_toponyms'"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# define the path to the folder where the data will be stored\n",
+    "data_path = \"../../../data_from_minio/wiener_diarum_toponyms\"\n",
+    "os.makedirs(data_path, exist_ok=True)\n",
+    "os.path.join(data_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|████████████████████████████████████████████| 1/1 [00:01<00:00,  1.19s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "files = [\"annotations_3-21_v4.html\"]\n",
+    "\n",
+    "\n",
+    "for file in tqdm(files):\n",
+    "    client.fget_object(\n",
+    "        bucket_name=\"knodle\",\n",
+    "        object_name=os.path.join(\"datasets/wiener_diarum_toponyms/\",file),\n",
+    "        file_path=os.path.join(data_path, file[-1]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Read in .html "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load html file\n",
+    "\n",
+    "with open(os.path.join(data_path, file[-1]), 'r', encoding = \"utf-8\" ) as f:\n",
```

Comment:
Don't use spaces around the "=" sign when used to indicate an argument. Also the space before the closing parenthesis should be removed.